cdk/lib/mobile-save-for-later.ts

```diff
@@ -39,7 +39,6 @@ export class MobileSaveForLater extends GuStack {
       runtime: Runtime.JAVA_8, // We should upgrade to Java 11 in a future PR
       app,
       fileName: `${app}.jar`,
-      // N.B. these lambdas previously used very specific, low memory sizes (384 & 394); odd choices for a Scala Lambda
```

Comment:
This was a reminder (to myself!) to mention this change when introducing the `cdk` version of this infrastructure. We discussed it [here](https://github.com/guardian/mobile-save-for-later/pull/65#discussion_r880284253), so this comment is no longer needed.